dragula.js

```diff
@@ -59,8 +60,13 @@ function dragula (initialContainers, options) {
 
   return drake;
 
-  function isContainer (el) {
-    return drake.containers.indexOf(el) !== -1 || o.isContainer(el);
+  function isContainer (el, handle) {
+    handle = handle || null;
```

Comment:
hi :) 👋 
This define is superfluous, as `handle` will be `undefined` if not passed in (or fail a bool check for a `falsy` value), hence, for readability, usability and performance, we could use an early return statement here as such:

```javascript
// There is both a handle and nested containers are allowed
if (handle && o.allowNestedContainers) {
  return drake.containers.indexOf(el) !== -1 || o.isContainer(el, handle);
}

return drake.containers.indexOf(el) !== -1 || o.isContainer(el);
```

Also, whenever modifying one of the arguments as such, it is best to redefine a variable in the local scope, in order to avoid accidental leaking in the future, through name changes and such small changes, where this approach is very bug prone:

```javascript
// This will leak into the global scope if the argument name changes, but the variable name doesn't
handle = handle || null;

// This will never leak into the global scope - redefined local variable references the original argument 
var handle = handle || null;
```

Many, many thanks for your open work in regards to this :) I am a bit disappointed the authors left it like that for over 1 year, it doesn't show much commitment at all. Thanks again!

Cheers